cli/cmd/version.go

```diff
@@ -11,15 +11,16 @@ import (
 
 // This variable is used by the "version" command and is set during build.
 // Defaults to a safe value if not set.
-var Version = "v0.0.2"
+var Version = "v0.0.5"
```

Comment:
Should we just dig this out of `runtime/debug` like we had talked about? Might be out of scope for this PR.

Reply:
I tried and couldn't get a version out of `runtime/debug` at all. There's probably a trick to it, but I think it's a future problem.

Reply:
Huh, interesting... I guess this works for now though.